tripper/triplestore.py

```diff
@@ -688,11 +381,10 @@ def add_function(
 
         method = getattr(self, f"_add_function_{standard}")
         func_iri = method(func, expects, returns, base_iri)
-        self.function_repo[func_iri] = func
-
+        if callable(func):
+            self.function_repo[func_iri] = func
```

Comment:
Before, this was always set - is there a point in the usage of an instance of this class that this fact was expected?

Reply:
Yes. In OntoFlow we want to semantically describe a model or conversion function without necessarily providing an implementation of it. To support that, `func` can either be a callable or just an IRI that stands for the model or function.

Reply:
Sorry, my question was not clear. I'll try again.
Prior to this change, the value `self.function_repo[func_iri]` was _always_ set. However, now it is _only_ set if `func` is a callable.
Is there any point in the code base, where this value might be expected to be present without actually performing a check for it? If yes, this value should probably be instantiated with `None` or set to `None` in an `else` block, in order to avoid a `KeyError` (and instead raise a more understandable `ValueError` or similar?).

Reply:
Not in tripper or OntoFlow. But if you try to instantiate a datamodel that via a missing mapping function DLite will fail...

Hmm, setting it to None is actually a good idea. Then DLite would be able to come with a better error message. 
Implemented!